doc/user-interface.md

```diff
@@ -0,0 +1,36 @@
+# User interface screen
+
+## Setting user interface
+
+The User interface screen is an optional part of an `auth` module dropdown of ASAB Web UI. It shows the information about version and build date.
+
+```
+User interface
+You can see the details of your user interface here
+
+Last build | 2021-07-27
+Version    | 8992ba2+dirty
+```
+
+### Setup
+
+Load the page by adding a route in `asab-webui/src/modules/auth/index.js`
+
+```javascript
+app.Router.addRoute({
+	path: '/auth/interface',
```

Comment:
The "User interface" card should be part of the About screen: a card below the existing card.
Not the dedicated router entry.